config.js

```diff
@@ -19,7 +19,7 @@ const defaultConfig = {
       'owners.name',
     ],
     attributesForFaceting: ['onlyFilter(name)', 'keywords'], // optionalFacetFilters to boost the name
-    customRanking: ['desc(downloadsLast30Days)'],
+    customRanking: ['desc(downloadsLast30Days)', 'desc(dependents)'],
```

Comment:
This will have no impact because desc(downloadsLast30Days) is already too precise so dependents would only be taken into account if downloads are the same.

What we could do is have a custom ranking that is made of silos (like group 1,2,3,4,5,6,7..) and then use dependents.

If I am wrong sorry I do not remember exactly what downloadsLast30Days includes.